spark/build.sbt

```diff
@@ -0,0 +1,9 @@
+test in assembly := {}
+
+assemblyMergeStrategy in assembly := {
+  case "reference.conf" => MergeStrategy.concat
+  case "application.conf" => MergeStrategy.concat
+  case "META-INF/MANIFEST.MF" => MergeStrategy.discard
+  case "META-INF\\MANIFEST.MF" => MergeStrategy.discard
+  case _ => MergeStrategy.first
+}
```

Comment:
I think these project-specific settings should just be added to the `Build.scala` subprojects specifically then. I explicitly felt pain about `build.sbt` files in some subprojects and not others containing pieces of data.

One idea to provide some separation would be to define the subproject settings in separate files under `project`, and import them into `Build.scala`. But really it's good to have all the build stuff in one place and not have danglers in the subproject folders.